### PR TITLE
Bug 1949928: Remove references to EOL ImageStream Tags

### PR DESCRIPTION
**Pull request by adambkaplan**
Removing tests which reference the following EOL imagestream tags:

- nodejs 10, 10-ubi7
- perl 5.26-el7 5.26
- postgresql 10-el7 10
- ruby 2.5-ubi7 2.5

### CODE REVIEW

**Review by adambkaplan — COMMENTED** [review]
/assign @gabemontero 

I think this covers us?

**Comment by gabemontero** [comment]
> /assign @gabemontero
> 
> I think this covers us?

proof will be in the e2e pudding but it seems sufficient ... certainly nothing bad stands out

going to

/lgtm

and let's pay attention to e2e-agnostic-cmd and e2e-gcp-builds

image-ecosystem was ran in the samples bump PR

**Comment by gabemontero** [comment]
/assign @soltysh 

for the cmd changes

**Comment by gabemontero** [comment]
@adambkaplan aside from perhaps revisiting the should e2e-aws-builds be optional in cluster-samples-operator, having at least an optional test for e2e-agnostic-cmd in that repo seems warranted as well

when I get out from customer case work I'll look into it, unless yourself or someone else from @openshift/openshift-team-build-api can beat me to it

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Comment by gabemontero** [comment]
`failed to acquire lease: resources not found ` on e2e-gcp-builds

**Comment by xiuwang** [comment]
/lgtm
label qe-approved

File: test/extended/testdata/bindata.go [hunk 1]
```diff
@@ -31064,11 +31042,9 @@ os::cmd::try_until_success 'oc get imagestreamtags nginx:latest'
...
 os::cmd::try_until_success 'oc get imagestreamtags nginx:1.16'
 os::cmd::try_until_success 'oc get imagestreamtags nodejs:latest'
-os::cmd::try_until_success 'oc get imagestreamtags nodejs:10'
 os::cmd::try_until_success 'oc get imagestreamtags nodejs:12'
 os::cmd::try_until_success 'oc get imagestreamtags perl:latest'
 os::cmd::try_until_success 'oc get imagestreamtags perl:5.30'
-os::cmd::try_until_success 'oc get imagestreamtags perl:5.26-ubi8'
 os::cmd::try_until_success 'oc get imagestreamtags php:latest'
 os::cmd::try_until_success 'oc get imagestreamtags php:7.0'
 os::cmd::try_until_success 'oc get imagestreamtags php:7.1'
```
Comment (xiuwang) [inline test/extended/testdata/bindata.go:31050]: php:7.0 and php:7.1 are EOL too

**Comment by gabemontero** [comment]
looks like we got a new regression @adambkaplan in the jenkins stuff .... saw the same thing in @vrutkovs 's OKD samples bump

on the plus side, it looks like all the EOL related stuff in your PR is good wrt e2e-gcp-builds

tagging @akram here but will ping them in their slack channel as well

**Comment by gabemontero** [comment]
Don't know if this is relevant, but the dump of the jenkins log shows

```
Exception in thread "main" java.lang.NoClassDefFoundError: org/acegisecurity/providers/encoding/ShaPasswordEncoder
	at com.redhat.openshift.PasswordEncoder.main(PasswordEncoder.java:20)
Caused by: java.lang.ClassNotFoundException: org.acegisecurity.providers.encoding.ShaPasswordEncoder
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:581)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 1 more
```

and requests to obtain info from jenkins are failing with authentication failures

**Comment by adambkaplan** [comment]
/hold cancel

The Jenkins regression has been reverted.

**Pull request merged**